int main()
{
	int num[2];
	num[2] = 2;	// exception
}